src/emitter.rs

```diff
@@ -300,9 +302,15 @@ fn need_quotes(string: &str) -> bool {
 
     string == ""
     || need_quotes_spaces(string)
+    || string.starts_with(|character: char| {
+        match character {
+            ':' | '&' | '*' | '?' | '|' | '-' | '<' | '>' | '=' | '!' | '%' | '@' => true,
```

Comment:
as mentioned in my comment: string may technically start with a `-` as long as they aren't followed by a space.

Reply:
i think this is good enough.